docker-compose-proxy.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.3'
```

Comment:
You're correct that Docker Compose is only intended for local development.  It's preferred to use Docker Swarm instead in production.